client/src/base-components/AppBar/AppBar.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { MuiAppBar, Toolbar, Box, Typography } from './AppBar.styled';
+
+export default function AppBar({
+  children,
+  color,
+  endCmp,
+  label,
+  position,
+  startCmp,
+  ...props
+}) {
+  return (
+    <MuiAppBar color={color} position={position} {...props}>
+      <Toolbar>
+        <Box>
+          {startCmp}
+          {label ? <Typography variant="h6">{label}</Typography> : null}
+        </Box>
+        <Box>
+          {children}
+          {endCmp}
```

Comment:
why need end component when you have children there?

Reply:
i understand, removed